validation/src/main/scala/hmda/validation/engine/lar/quality/LarQualityEngine.scala

```diff
@@ -7,6 +7,11 @@ import hmda.validation.rules.lar.quality._
 
 trait LarQualityEngine extends LarCommonEngine with ValidationApi {
 
+  private def q022(lar: LoanApplicationRegister): LarValidation = {
+    // TODO: 2017 is hardcoded for now.  It will be passed in through the API at a later date
```

Comment:
Also, check out the discussion on the issue ticket.  It includes most of what I just wrote, but it might have more helpful information as well.